src/lib.rs

```diff
@@ -126,16 +149,25 @@ where
 
     /// Read humidity and temperature.
     pub fn read(&mut self) -> Result<(Humidity, Temperature), Error<E>> {
-        let buf: &mut [u8; 7] = &mut [0; 7];
-        // Sort of reverse engineered the cmd data:
-        // Bit 0 -> temperature calibration (0 => +0.5C)
-        // Bit {1,2,3} -> refresh rate? (0 => slow refresh)
         self.i2c
-            .write_read(I2C_ADDRESS, &[Command::GetCT as u8, 0b11111111, 0], buf)?;
-        let status = StatusFlags { bits: buf[0] };
-        if !status.contains(StatusFlags::CALIBRATION_ENABLE) {
-            return Err(Error::Uncalibrated());
+            .write(I2C_ADDRESS, &[Command::GetCT as u8, 0x33, 0])?;
```

Comment:
I also changed this constant based on what adafruit example had. They use 0x33 instead of 0xFF.

I could not find any good datasheet documentation of what one does vs the other. Based on the previous code comment it seems no good reference exists in general.